bundles/pubsub/pubsub_admin_tcp/src/pubsub_tcp_topic_receiver.c

```diff
@@ -150,6 +150,8 @@ pubsub_tcp_topic_receiver_t *pubsub_tcpTopicReceiver_create(celix_bundle_context
     pubsub_tcp_topic_receiver_t *receiver = calloc(1, sizeof(*receiver));
     receiver->ctx = ctx;
     receiver->logHelper = logHelper;
+
+    L_WARN("created receiver %p", receiver);
```

Comment:
Isn't L_INFO better here? Not sure what the general usage of WARN vs INFO is...

Reply:
Debug stuff, I'll remove it